utils/dbt_format_utils.py

```diff
@@ -0,0 +1,329 @@
+        # Check Args for Action
+        if not args.basePath:
+            raise Exception("Make sure you specify the --basePath argument")
+        if not args.outputPath:
+            raise Exception("Make sure you specify the --outputPath argument")
+        # Run selected action
+        create_matview_dbt_files_from_base(args.basePath, args.outputPath)
+
+    elif args.action == "getCacherScript":
+        # Check Args for Action
+        if not args.cacherSnippetGUID:
+            raise Exception("Make sure you specify the --cacherSnippetGUID argument")
+        if not args.cacherApiKey:
+            raise Exception("Make sure you specify the --cacherApiKey argument")
+        if not args.cacherApiToken:
+            raise Exception("Make sure you specify the --cacherApiToken argument")
+        if not args.outputPath:
+            raise Exception("Make sure you specify the --outputPath argument")
+        # Run selected action
+        download_cacher_script(
+            args.cacherSnippetGUID,
+            args.cacherApiKey,
+            args.cacherApiToken,
+            args.outputPath,
+        )
+
+    elif args.action == "restructAirbyteDbt":
+        # Check Args for Action
+        if not args.airbyteWorkspacePath:
+            raise Exception("Make sure you specify the --airbyteWorkspacePath argument")
+        if not args.dbtCtaPath:
+            raise Exception("Make sure you specify the --dbtCtaPath argument")
+        # Run selected action
+        restructure_airbyte_dbt(args.airbyteWorkspacePath, args.dbtCtaPath)
+    else:
+        raise Exception(
+            "Please enter a supported action -> renameBaseTables, createMatViews, getCacherScript, restructAirbyteDbt"
+        )
+
+
+# Function to grab Airbyte Generated dbt and move into CTA structured folder (✨Progify it✨)
+def restructure_airbyte_dbt(airbyte_workspace_path, dbt_cta_path):
+    # Mapping of Airbyte folders -> CTA folders
+    airbyte_cta_dir_mapping = {
+        "airbyte_ctes": "0_ctes",
+        "airbyte_tables": "1_cta_full_refresh",
+        "airbyte_incremental": "1_cta_incremental",
+    }
+
+    # Go through and move all files from Airbyte export into 'dbt-cta' directories
+    for airbyte_dir, cta_dir in airbyte_cta_dir_mapping.items():
+        # Get all sql files that should be moved for current mapping
+        files_to_move = glob(
+            f"{airbyte_workspace_path}/**/{airbyte_dir}/**/*.sql", recursive=True
+        )
+
+        if files_to_move:  # Only run if there were any files found
+            # Make sure target path is created
+            dbt_cta_path = (
+                dbt_cta_path if dbt_cta_path[-1] != "/" else dbt_cta_path[:-1]
+            )
+            target_path = f"{dbt_cta_path}/models/{cta_dir}"
+            os.makedirs(target_path, exist_ok=True)
+
+            # Move files to target path
+            for file_path in files_to_move:
+                file_name = file_path.split("/")[-1]
```

Comment:
I have recently learned through the magic of ChatGPT that a slightly better practice here is to use `os.path.basename()` instead if splitting the path string! I guess it's more platform independent, and it's unclear if that's actually a relevant concern, but bp's are nice 👀 

```suggestion
                file_name = os.path.basename(file_path)
```

In general I highly recommend asking ChatGPT for bp's regarding file operations - there's a lot you can do more cleanly with `os` (this is all news to me via my own heftychonk PR w the Cloud Function).